SlevomatCodingStandard/Sniffs/ControlStructures/AbstractControlStructureSpacingSniff.php

```diff
@@ -206,7 +213,12 @@ protected function checkLinesAfter(File $phpcsFile, int $controlStructurePointer
 	{
 		$tokens = $phpcsFile->getTokens();
 
-		$controlStructureEndPointer = $this->findControlStructureEnd($phpcsFile, $controlStructurePointer);
+		try {
```

Comment:
what is the point of try-catch here, if it just rethrows the exception?

Reply:
Yeah, I was lazy to again write review. I plan to remove it in master.

Reply:
From my point of view, I did it that way to make the code self-documenting, making it extremely clear that the exception is passed on.